docs/_includes/home-flow-item.html

```diff
@@ -1,12 +1,17 @@
 {% for item in include.data %}
-  <div class="headline">{{ item.headline }}</div>
-  <div class="flowContainer{% if item.imagepos == 'left' %} imageLeft{% else %} imageRight{% endif %}">
+  <div id="flowContainer{{forloop.index}}" class="FlowContainer{% if item.imagepos == 'left' %} imageLeft{% else %} imageRight{% endif %}">
+    <svg class="dashedLine" width="50" height="880px" viewBox="0 0 50 880">
```

Comment:
I have the container that it's in (.FlowContainer) set as overflow:hidden, with the dashedLine set to 880 (the longest it can possibly be on the page). Let me know what you'd suggest.